xblockutils/public/studio_edit.js

```diff
@@ -66,7 +65,9 @@ function StudioEditableXBlockMixin(runtime, element) {
                 setup : function(ed) {
                     ed.on('change', fieldChanged);
                 }
-            });
+            };
+            tinyMCE.baseURL = baseUrl + "/js/vendor/tinymce/js/tinymce";
+            $field.tinymce(Object.assign(tinymceOptions, window.CustomizeFunctionsHook.getTinymceExtraOptions()));
```

Comment:
Hi @tranphan-groove ,

Check window.CustomizeFunctionsHook and window.CustomizeFunctionsHook.getTinymceExtraOptions before using it